www/scripts/main.js

```diff
@@ -229,6 +230,11 @@ function setLastRoomName(roomName) {
 
 /* Helpers */
 
+function displayMessage(recipient) {
```

Comment:
My grasp of tone isn't statically typed either so I've interpreted your reply as positive.